dsaas-services/f8-toggles-service.yaml

```diff
@@ -0,0 +1,5 @@
+services:
+- hash: 7f988ff0c74ccbfb0cd1e56d04f698d0ff55be4d
+  name: fabric8-toggles
```

Comment:
name has to be fabric8-toggles-service (matching the repo name for convenience in CI/CD)

Reply:
oh, I missed that O_O 
thanks, @vpavlin !